pkg/wait/wait_time_test.go

```diff
@@ -57,7 +57,7 @@ func TestWaitTestStress(t *testing.T) {
 	for i := 0; i < 10000; i++ {
 		chs = append(chs, wt.Wait(uint64(i)))
 	}
-	wt.Trigger(10000 + 1)
+	wt.Trigger(10000 - 1)
```

Comment:
Do we need the math operation here?  Could we just change the value to `9999`? Not a blocker it just took me a moment to review what the test was trying to do.

Reply:
something like below better?
```
	for i := 0; i <= 9999; i++ {
		chs = append(chs, wt.Wait(uint64(i)))
	}
	wt.Trigger(9999)
```

Reply:
Agreed this is clearest, I'll update the MR.

Reply:
Updated indices in tests.